features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -24,7 +24,7 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
     Then I should see "Edit first fandom Tag"
     
     # assigning media to a fandom
-    When I fill in "Media" with "TV Shows"
+    When I fill in "tag[media_string]" with "TV Shows"
```

Comment:
Just wondering why this needs to change. Also, as Ariana mentioned in our discussion, we're definitely going to want some tests to make sure that caching is working right.